README.md

```diff
@@ -61,7 +61,7 @@ Running the `serve` command in your app directory will start your local developm
 $ shopify serve
 ```
 
-Your terminal will display the localhost and port where your app is now visible.
+Your terminal will display the localhost and port where your app is now visible. Your app will then be able to be installed on a Dev Store from your Partner Dashboard.
```

Comment:
`will then be able to be` is a bit of a mouthful. How about simply `Your app can then be installed...`

`Dev Store` can just be `development store`